Raven.Database/Raft/ClusterManagementHttpClient.cs

```diff
@@ -345,7 +345,20 @@ public Task SendNodeUpdateAsync(NodeConnectionInfo node)
                 }
                 else
                 {
+                    // before we send remove from cluster wait until configuration will be propaged
```

Comment:
I will try to spin for first condition with some timeout. And review whether we can eliminate second hard wait.